### DISCUSSION TITLE
715

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#897] [Comments]
# 715

3년차가 안되던 주니어 개발자일때를 돌이켜보면 "과연 내가 이 팀에서 필요한 사람인가" 하는 의문이 들때가 많았다. 당시에 내가 속해있던 팀은 연차에 관계 없이 잘하는 사람들이 많았다. 나보다도 어린, 나보다도 연차가 낮은 개발자분들이 나보다 훨씬 더 좋은 코드로 훨씬 더 빠른 속도로 일을 마무리 하는 것을 볼때마다 계속해서 나와 비교를 했다. 좋은 팀을 이야기할때 "개인이 풀스택일 필요는 없지만, 팀은 풀스택이어야 한다"는 말을 자주 듣는다. 한 사람이 모든 영역을 잘할 수 없기 때문에, 각자가 고유의 전문 영역을 가지고 서로의 부족함을 채워준다는 의미로 해석했다. 하지만, 이 팀에서 내가 무엇을 채워줄 수 있을지, 무엇이 내 고유의 전문영역인지 알 수가 없었다. 당시의 우리팀은 퍼블리싱을 제외한 모든..

https://jojoldu.tistory.com/715

**Comment by YohanLee96** [comment]
슬럼프를 남을 돕는 행동으로 잘 승화시키셨네요ㅎㅎ 매번 좋은글 잘보고갑니다.

**Comment by jdongchan** [comment]
저도 책 '타이탄의 도구들'의 '안테암불로가 되어라' 대목을 좋아하면서도, 남이나 잠재적 경쟁자를 돕기 전에, 항상 상황이나 이익을 재게 되더라고요. 향로님께서 좋은 사례를 통해 제 가치관을 조금 더 일관되게 해주셔서 감사하네요!
